docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '3'
 services:
   app:
-    image: decidim/decidim:0.24.3
+    image: decidim/decidim:0.25.2
```

Comment:
Ups, I see no 0.25 docker image in the Docker Hub. Anyway, it will be updated in the next PR with the 0.26 and 0.27 updates.

I will rollback it to the 0.24 image